src/main/scala/scalaz/ziokeeper/crdt/ORSet.scala

```diff
@@ -0,0 +1,31 @@
+package scalaz.ziokeeper.crdt
+
+import scalaz.ziokeeper.VectorClock
+
+case class ORSet[A, B](values: Vector[ORSet.VersionedEntry[A, B]]) {
```

Comment:
This is gonna be super slow. But I think that's OK. We should try to make it correct first, and we can optimize later. 👍